### PR TITLE
Add links to related projects

### PR DESCRIPTION
**Pull request by felixLam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@
 
 **A floating `UIToolBar` replacement as seen in the iOS 10 Maps app, supporting both vertical and horizontal orientation.**  
 
-It is designed to hover above your content and it plays nicely with auto layout.
+It is designed to hover above your content and it plays nicely with auto layout. Combine
```
Comment (shagedorn) [inline README.md:9]: `resemble` instead of `recreate`?

File: README.md [hunk 2]
```diff
@@ -113,6 +115,10 @@ github iosphere/ISHHoverBar
 See the [Carthage repository](https://github.com/Carthage/Carthage) to get started
 with Carthage.
 
+## More OpenSource projects by iosphere
+
+* [`ISHPermissionKit`](https://github.com/iosphere/ISHPermissionKit) - A polite and unified way of asking for permission on iOS
```
Comment (shagedorn) [inline README.md]: Can we display the icons here, similar to the "Apps using permission kit" section?

File: README.md (hunk 1 above)
Comment (felixLam) [inline README.md:9]: fixed

File: README.md (hunk 2 above)
Comment (felixLam) [inline README.md]: Fixed

**Comment by shagedorn** [comment]
👍

**Pull request merged**